.circleci/config.yml

```diff
@@ -77,3 +84,13 @@ workflows:
               ignore: /.*/
             tags:
               only: /[0-9]+\.[0-9]+\.[0-9]+/
+  scheduled-integration-tests:
+    triggers:
+      - schedule:
+          cron: "*/7 * * * *"
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
Can you temporarily change this to the current branch `schedule-it-circleci` and also adjust the Cron schedule for testing before merging it?

Reply:
yeah, good idea, let me try that. I think I tried without the filter and it didn't work either but maybe that works.